code/modules/mob/living/simple_animal/friendly/crab.dm

```diff
@@ -43,4 +43,35 @@
 	response_help  = "pets"
 	response_disarm = "gently pushes aside"
 	response_harm   = "stomps"
-	gold_core_spawnable = 0
\ No newline at end of file
+	gold_core_spawnable = FALSE
+
+/mob/living/simple_animal/crab/evil
+	name = "Evil Crab"
+	real_name = "Evil Crab"
+	desc = "Unnerving, isn't it? It has to be planning something nefarious..."
+	icon_state = "evilcrab"
+	icon_living = "evilcrab"
+	icon_dead = "evilcrab_dead"
+	response_help = "pokes"
+	response_disarm = "shoves"
+	response_harm = "stomps"
+	gold_core_spawnable = TRUE
+
+/mob/living/simple_animal/crab/kreb
+	name = "Kreb"
+	desc = "This is a real crab. The other crabs are simply gubbucks in disguise!

```

Comment:
...disguise!" your forgot the BLIMMIN' QUOTATION MARK, CAN YOU NOT READ COMPILE LOGS ARM!!! ARM!!! AAAAAAAAAARM!!!!!!!!